client/src/components/Profile.jsx

```diff
@@ -95,26 +98,29 @@ function Profile({
     fetchReviewsWithTitles();
   }, [userData, baseURL]);
 
-  // Delete a user's review
-  const deleteUserReview = async (reviewId) => {
-    try {
-        if (!loggedInUser?.user_id || !reviewId) {
-            console.log("Missing user_id or review_id", loggedInUser.user_id, reviewId);
-            return;
-        }
+  console.log("USER DATA", userData)
+  console.log("but this is the basic user data", loggedInUser)
 
-        const response = await axios.delete(`${baseURL}/reviews/${loggedInUser.user_id}/${reviewId}`
-        );
-        if (response.status === 200) {
-            setReviewsWithGameTitles((prevReviews) =>
-            prevReviews.filter((review) => review.review_id !== reviewId)
-        );
-        console.log("Review deleted successfully");
-        }
-    } catch (error) {
-        console.error("Error deleting review", error);
-    }
-  };
+  // Delete a user's review
+//   const deleteUserReview = async (reviewId) => {
+//     try {
+//         if (!loggedInUser?.user_id || !reviewId) {
+//             console.log("Missing user_id or review_id", loggedInUser.user_id, reviewId);
+//             return;
+//         }
+
+//         const response = await axios.delete(`${baseURL}/reviews/${loggedInUser.user_id}/${reviewId}`
+//         );
+//         if (response.status === 200) {
+//             setReviewsWithGameTitles((prevReviews) =>
+//             prevReviews.filter((review) => review.review_id !== reviewId)
+//         );
+//         console.log("Review deleted successfully");
+//         }
+//     } catch (error) {
+//         console.error("Error deleting review", error);
+//     }
+//   };
```

Comment:
I see this is where it was before. Is this comment still here in case you want to move it back?